torch/_jit_internal.py

```diff
@@ -343,7 +344,8 @@ def get_annotation_str(annotation):
         # In Python3.9+ subscript indicies are not wrapped in ast.Index
         subscript_slice = annotation.slice if sys.version_info >= (3, 9) else annotation.slice.value  # type: ignore[attr-defined]
         return f"{get_annotation_str(annotation.value)}[{get_annotation_str(subscript_slice)}]"
-    elif isinstance(annotation, ast.Tuple):
+    elif isinstance(annotation, ast.Tuple) or isinstance(annotation, ast.List):
+        strs = [get_annotation_str(elt) for elt in annotation.elts]
```

Comment:
if this isn't necessary for this PR can we separate it out?

had some questions here:
- (nit) i think the strs variable isn't being used anywhere?
- is comma-separated list of contained-types appropriate for a list type?
- also can we add a test for the previously failing behavior?

Reply:
Yes, we do not need this change - will drop it.
I think I just left it from debug session.